srv/components/csm/config/config.sls

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+include:
+    - components.csm.config.post_install
```

Comment:
@tanujashinde0405 I think we're calling all phases in `init_mod` . Please follow either of this - 
1. include the flow of all phases in init, or 
2. include each phase inside the next phase's file. (based on `require` steps)

Overall, better to follow step-1, to maintain uniformity and file discipline.